src/context.rs

```diff
@@ -58,4 +63,23 @@ impl Context {
   ) -> Local<'s, Object> {
     unsafe { scope.cast_local(|_| v8__Context__Global(self)) }.unwrap()
   }
+
+  /// Return data that was previously attached to the context snapshot via
+  /// SnapshotCreator, and removes the reference to it.
+  /// Repeated call with the same index return None.
+  pub unsafe fn get_data_from_snapshot_once<'s, T>(
+    &self,
+    scope: &mut HandleScope<'s, ()>,
```

Comment:
Remove the `()` param. This means that this HandleScope is *not* in a context.
You want to opposite.